core/assets/bundles/bundle.properties

```diff
@@ -705,22 +705,22 @@ planet.sun.name = Sun
 
 sector.impact0078.name = Impact 0078
 sector.groundZero.name = Ground Zero
-sector.craters.name = The Craters
-sector.frozenForest.name = Frozen Forest
-sector.ruinousShores.name = Ruinous Shores
-sector.stainedMountains.name = Stained Mountains
-sector.desolateRift.name = Desolate Rift
-sector.nuclearComplex.name = Nuclear Production Complex
-sector.overgrowth.name = Overgrowth
-sector.tarFields.name = Tar Fields
-sector.saltFlats.name = Salt Flats
-sector.fungalPass.name = Fungal Pass
-sector.biomassFacility.name = Biomass Synthesis Facility
-sector.windsweptIslands.name = Windswept Islands
-sector.extractionOutpost.name = Extraction Outpost
-sector.planetaryTerminal.name = Planetary Launch Terminal
-sector.coastline.name = Coastline
-sector.navalFortress.name = Naval Fortress
+sector.craters.name = Die Krater
+sector.frozenForest.name = Gefrorener Wald
+sector.ruinousShores.name = Trümmerstrände
+sector.stainedMountains.name = Geflecktes Gebirge
+sector.desolateRift.name = Einsame Kluft
+sector.nuclearComplex.name = Kernherstellungskomplex
+sector.overgrowth.name = Überwucherung
+sector.tarFields.name = Teerfelder
+sector.saltFlats.name = Salzebenen
+sector.fungalPass.name = Infizierter Gebirgspass
```

Comment:
That's not even the correct translation of fungal in this context

Reply:
I'd be open to suggestions, do you have a better idea?